### PR TITLE
ignoreMatches works on missing dependencies. Fixes depcheck/depcheck#190

### PR DESCRIPTION
**Pull request by goloroden**
Hi @lijunle!

It's me again ;-)
I have added a fix for #190, and I would love it if you could please merge it into the project!

All the checks were run successfully, and I've also added a test.

Have a nice day,

Golo

### CODE REVIEW

**Comment by lijunle** [comment]
Looks good, I could like to push to avoid `postProcessedResults` variable and make function pure. Then I will merge it.

File: src/index.js [hunk 1]
```diff
@@ -66,7 +66,14 @@ export default function depcheck(rootDir, options, callback) {
     optionalDeps,
     parsers,
     detectors,
-  }).then(callback);
+  })
+  .then(results => Object.assign(results, {
+    missing: lodash.pick(
+      results.missing,
+      filterDependencies(rootDir, ignoreBinPackage, ignoreMatches, results.missing),
```
Comment (lijunle) [inline src/index.js]: I am surprise that, this trailing comma is passing node v0.12 test.

**Comment by lijunle** [comment]
Fixes #190 #169

**Comment by lijunle** [comment]
Version 0.6.7 is out! Thank you @goloroden !

BTW, do you want to be a member of depcheck org? I could like to invite you! 😸

**Comment by goloroden** [comment]
This is great news 😊!

Thanks for merging, thanks for the new version, and thanks especially for the invitation 😊!

**Pull request merged**